tests/net/traffic_class/src/main.c

```diff
@@ -11,7 +11,7 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(net_test, NET_LOG_LEVEL);
 
-#include <zephyr/types.h>
+#include <zephyr/types.h>z
```

Comment:
look like this is an extra typo
and this? I suppose,  breaks compilation

Reply:
Fixed. thanks